<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateOptionDialogWidget</class>
 <widget class="QWidget" name="DateOptionDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>522</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,10">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Date format:</string>
       </property>
       <property name="buddy">
        <cstring>dateFormatPicker</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="dateFormatPicker"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KLineEdit" name="customFormatInput">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="exampleLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string comment="KDE::DoNotExtract">TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dateFormatLink">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string comment="KDE::DoNotExtract">TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
         <string comment="source to read date information from">Source</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="fromImageBtn">
        <property name="text">
         <string>From Image</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fixedDateBtn">
        <property name="text">
         <string>Fixed Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="fixedDateContainer" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="KDatePicker" name="datePicker"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Time:</string>
             </property>
             <property name="buddy">
              <cstring>timePicker</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="timePicker"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KDatePicker</class>
   <extends>QFrame</extends>
   <header>kdatepicker.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dateFormatPicker</tabstop>
  <tabstop>customFormatInput</tabstop>
  <tabstop>fromImageBtn</tabstop>
  <tabstop>fixedDateBtn</tabstop>
  <tabstop>timePicker</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>fixedDateBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>fixedDateContainer</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
